src/Firehose.Web/Authors/LuisBeltran.cs

```diff
@@ -0,0 +1,37 @@
+using Firehose.Web.Infrastructure;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using System.Web;
+
+namespace Firehose.Web.Authors
+{
+  public class LuisBeltran : IAmAMicrosoftMVP, IFilterMyBlogPosts
+  {
+      public string FirstName => "Luis";
+      public string LastName => "Beltran";
+      public string ShortBioOrTagLine => "is a passionate Mexican software community guy who enjoys learning, talking and helping others about Xamarin, Azure, and C#";
```

Comment:
"is a passionate Mexican software community guy, who enjoys teaching, talking and helping others about Xamarin, Azure, and C#."

Is more correct English.